internal/api/core/routes.go

```diff
@@ -9,6 +9,10 @@ import (
 
 var CURRENT_PREFIX string = fmt.Sprintf("/api/v%02d", util.MustGetAPIVersion())
 
+// API routes will be empty until RunServer() is called.
+// This is a cached version of api.GetRoutes() for packages that cannot access `internal/api` due to import cycles.
+var apiRoutes []Route
```

Comment:
Explicitly initialize this as nil.
It is not strictly necessary, but it's nice to do.